CodeSniffer/Standards/Generic/Sniffs/PHP/DisallowShortOpenTagSniff.php

```diff
@@ -72,7 +72,12 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
             $data  = array($token['content']);
             $fix   = $phpcsFile->addFixableError($error, $stackPtr, 'Found', $data);
             if ($fix === true) {
-                $phpcsFile->fixer->replaceToken($stackPtr, '<?php');
+                $correctOpening = '<?php';
+                if (isset($tokens[$stackPtr + 1]) && $tokens[$stackPtr + 1]['code'] !== T_WHITESPACE) {
```

Comment:
You'll need to make the `isset($tokens[$stackPtr + 1])` explicitely test against `true` for the build to pass:
```if (isset($tokens[$stackPtr + 1]) === true && ...)```